### ISSUE TITLE
Error finding GPT4

### ISSUE THREAD

**Issue by LayerMaker** [BerriAI/litellm#506]
line 775, in _interpret_response_line
    raise self.handle_error_response(
openai.error.InvalidRequestError: The model `gpt-4` does not exist or you do not have access to it. Learn more: https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4.
 
I have a pro GPT account and have created API Keys in the past, i do not know why it can not find my account or it has been dined access to my gpt account.

Some insight would be most appreciated

**Comment by ishaan-jaff** [comment]
@LayerMaker looks like you don't have access to GPT-4. Can I see how you're making the litellm completion call ?

**Comment by ghost** [comment]
@LayerMaker I believe your referring to chatgpt plus. This is not the same as the OpenAI API. This might be helpful - https://help.openai.com/en/articles/7039783-how-can-i-access-the-chatgpt-api

**Comment by ghost** [comment]
Closing for now as this is not a litellm issue but user access problem. @LayerMaker feel free to reopen if this ends up not being the case.

**Comment by LayerMaker** [comment]
Hi KishI worked it out eventually,I worked out that to use the api and make a call to Chat GTP you need credits, additional to your paid chat GTP subscription.Git-Hub gave me that answers I was looking for by taking the errors in terminal and pasting them into a search engine.I have a simple suggestion if I may be so bold as to offer it.On the open Ai page where this is addressed, why don’t you put in plain English that if you are trying to use GTP locally and you are getting “can not connect to GTP 4” error run back in your terminal, that this will require credits outside of a paid subscription to chat GTP. With a link to top up your open Ai credits. Where I toped up mine.It would save a lot of hassle, not everyone is in the same intelligence bracket as you guys that are working with machine learning.And Chat GTP can’t browse the internet to find out and explain it to our dumb asses lol.Anyway it’s just a suggestion, maybe I’m being stupid and it was clear in the text provided.Have a good rest of your week.Kind Regards,Charlie Sent from my iPhoneOn 2 Oct 2023, at 19:06, Krish Dholakia ***@***.***> wrote:﻿
@LayerMaker I believe your referring to chatgpt plus. This is not the same as the OpenAI API. This might be helpful - https://help.openai.com/en/articles/7039783-how-can-i-access-the-chatgpt-api
